core/packages/test/scripts/build-binary.sh

```diff
@@ -72,7 +72,6 @@ build_e2e_test() {
     pnpm install
     popd
     pushd "$contract_dir"
-    forge install
```

Comment:
@yrong Don't we need this to [ensure that dependencies are available](https://book.getfoundry.sh/projects/working-on-an-existing-project)? New and clean repos fail to run the forge deploy script without this.